### DISCUSSION TITLE
Unitypackage?

### DISCUSSION THREAD

**Discussion by ttkrpink** [fo-dicom/fo-dicom#1245] [Q&A]
**What is, according to you, missing from fo-dicom? Please describe.**
The last fo-dicom unity asset store package was almost 4 years ago. The latest windows MRTK does not like the old fo-dicom version.

**Describe the solution you'd like**
A unitypackage file that one can download and start using, working with Windows SDK 19041, because there are features in the Mixed Reality Took Kit 2.7 and working with Hololens 2.

**Describe possible alternatives or existing workarounds you've considered**
can't think of any.

**Comment by Kent1LG** [comment]
Hi,

I am 100% for this idea !
Meanwhile, is there any way to implement fo-dicom on Unity for HoloLens?
I found that there is a Unity (even HoloLens) solution on the repository, but I don't know how to use it...
Moreover, I tried to build the VS solution (DICOM.Full.sln and DICOM.Unity.sln), but I'm facing many errors.

**Comment by mrbean-bremen** [comment]
fo-dicom 5 targets NetStandard 2.0, which should be compattible to Unity, as far as I understand.
@gofal - is that correct?

**Comment by gofal** [comment]
Yes, netstandard 2.0 can be used in newer unity versions. If you search the issues for "unity" then you will find some hints about how to use unity.
For example the unity IL2CPP does some code optimizations via reflection and omits every class that is not touched at compiletime. 
Or there are implementations of IImage that return you unity compatible images. Anders Gustaffson had implemented it and hosted it on some package storage for unity and then added the code here in github issues to make it available for anyone interested.

**Comment by JHsGame** [comment]
@ttkrpink 

Have you had any success fo-Dicom update in Unity?

Please let me know if the update was successful.

I want to update fo-Dicom in Unity because it is essential for what I do.

**Comment by Darkmax** [comment]
Also im interested on how to compile it in unity for hololens 2, I tried but without success
